README.md

```diff
@@ -44,7 +44,7 @@ Step 4: Install the Home Assistant Supervised Debian Package:
 
 ```bash
 wget https://github.com/home-assistant/supervised-installer/releases/latest/download/homeassistant-supervised.deb
-apt install ./homeassistant-supervised.deb
+dpkg -i ./homeassistant-supervised.deb
```

Comment:
Hi there @AnonTester 👋 

Thanks for the PR!
Could you motivate why this change is needed?
It is missing from your PR description.

Thanks! 👍 

../Frenck